cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackend.java

```diff
@@ -75,196 +67,207 @@ public void close(){
     } // close
 
     /**
-     * Sets the MySQL driver. It is protected since it is only used by the
+     * Sets the SQL driver. It is protected since it is only used by the
      * tests.
-     * 
-     * @param driver
-     *            The MySQL driver to be set.
+     *
+     * @param driver The SQL driver to be set.
      */
-    protected void setDriver(MySQLDriver driver) {
+    public void setDriver(SQLBackend.SQLDriver driver) {
         this.driver = driver;
     } // setDriver
 
-    protected MySQLDriver getDriver() {
+    public SQLBackend.SQLDriver getDriver() {
         return driver;
     } // getDriver
 
-    @Override
-    public void createDatabase(String dbName) throws CygnusRuntimeError, CygnusPersistenceError {
-        if (cache.isCachedDb(dbName)) {
-            LOGGER.debug("'" + dbName + "' is cached, thus it is not created");
+    public void createDestination(String destination) throws CygnusRuntimeError, CygnusPersistenceError {
+        if (cache.isCachedDestination(destination)) {
+            LOGGER.debug("'" + destination + "' is cached, thus it is not created");
```

Comment:
I guess all logs should include instance name

Reply:
Ok. The class has a generic name so it can not be changed. Is it ok if I add the instance as a sufix of the class name? For instance:

com.telefonica.iot.cygnus.sql.sqlBackend.mysql

Would that be ok?

Reply:
It would be ok, something know what instance (mysql or postgresql/postgis)

Reply:
Maybe include self.sqlInstance value in SQLBackendImpl logs would be enough

Reply:
great idea! Thanks @AlvaroVega.

Fixed in [73fcd16](https://github.com/telefonicaid/fiware-cygnus/pull/1842/commits/73fcd169075d36e70b889d53fbf82c37dbb1bcba)